core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -665,13 +665,15 @@ public void createWithoutOverwrite() throws Exception {
     when(alluxioFs.exists(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))))
         .thenReturn(true);
     when(alluxioFs.createFile(eq(new AlluxioURI(HadoopUtils.getPathWithoutScheme(path))), any()))
-        .thenThrow(new FileAlreadyExistsException(path.toString()));
+        .thenThrow(new FileAlreadyExistsException("Not allowed to create() (overwrite=false) for "
```

Comment:
some repetitive strings here, let's use `ExceptionMessage` and `Message.format()`?

Reply:
What about "Alluxio path {} exists and create(overwrite=false)"?
Actually i'm not so sure how to express `create(overwrite=false)` better in java (it's too easy in python). Feel free to propose a better one :)

Reply:
fix some error messages